ppr-ui/src/interfaces/mhr-registration-interfaces/MhrRegistrationHomeLocationIF.ts

```diff
@@ -1,8 +1,8 @@
 import { HomeLocationTypes } from '@/enums'
 
 export interface MhrRegistrationHomeLocationIF {
```

Comment:
Just for clarification, why set _almost_ all fields to optional?

Reply:
Definitely!

Most of the home location properties are optional to begin with, this goes back to the MH Registration itself. They could have been made optional then, when first implemented. I have done so now so we could utilize the same interface, for the same model, in the different flow. 
But yea, they are all optional, only some are ever required and that is really dependent on the type of selection the user makes for home location type.

Reply:
I see. The interfaces should help us with the development, to enforce type safety and hopefully have less issues. With all fields optional, yes it makes more flexible/reusable, but it's harder to catch potential errors. 
Should we make it more strict and/or create another IF or keep as is? Thoughts?

Reply:
I understand the purpose of interfaces. 

It's about as strict as it should be i think. If you reference the api schema, its of similar design. ie some required, some not. 
If you want to create an individual interface for every data permutation, have at er, but you will still end up with optional properties where the properties are... optional. 

I would argue the interface does it job well enough in informing us what should/could be in this data set. 
If i was to create another interface it would look near identical because this data set has a lot of optional properties.